apps/fluent-tester/src/TestComponents/Avatar/BasicAvatar.android.tsx

```diff
@@ -0,0 +1,243 @@
+import React, { FunctionComponent } from 'react';
+import { Avatar } from '@fluentui-react-native/avatar';
+import { View } from 'react-native';
+import { steveBallmerPhotoUrl } from './../PersonaCoin/styles';
+
+export const StandardUsage: FunctionComponent = () => {
+  return (
+    <View style={{ paddingBottom: 40 }} >
+      <View style={{
+        flexDirection: 'row',
+        marginHorizontal: 10,
+        paddingTop: 10,
+        paddingBottom: 5,
+        flex: 1,
+        justifyContent: 'space-evenly',
+      }}>
+
+        {/* Variation - Standard */}
+
+        {/* With badge. */}
+        <Avatar
+          badge={{ status: 'available' }}
```

Comment:
Please make this a const object or memoize it.

Reply:
Updated to const styles.

Reply:
I see, this was  not for styles but for the component props for badge as per @valentynas comment. I would take a note to resolve this comment in next PR's.